backend/src/main/java/hu/flowacademy/worksheet/controller/UserController.java

```diff
@@ -42,4 +43,10 @@ public User createUser(@RequestBody UserOperationDTO userOperationDTO) throws Va
     public AccessTokenResponse login(@RequestBody UserOperationDTO userOperationDTO) {
         return keycloakClientService.login(userOperationDTO.getEmail(), userOperationDTO.getPassword());
     }
+
+    @GetMapping("/users")
+    @ResponseStatus(HttpStatus.FOUND)
+    public List<User> findUserByEmailFirstNameLastName(@RequestParam(value = "q") String q) {
```

Comment:
Ez a művelet valamilyen felhasználói jogkörhöz kötött, nem? Mondjuk csak ADMIN tudja végezni. Ha így van, akkor legyen egy `@RolesAllowed(...)` is fölötte.